sumpy/fmm.py

```diff
@@ -674,7 +682,8 @@ def eval_direct(self, target_boxes, source_box_starts,
                 source_box_lists=source_box_lists,
                 strength=src_weight_vecs,
                 result=pot,
-
+                max_npoints_in_one_box=max(self.get_max_nsources_in_one_box(queue),
+                    self.get_max_ntargets_in_one_box(queue)),
```

Comment:
Not sure I'm a fan of this being recomputed on an ongoing basis...

Reply:
Fixed in 447e5eb